proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -130,6 +130,37 @@ service Microgrid {
     };
   }
 
+  // Adds inclusion bounds for a given metric of a given component, and returns
+  // the UTC timestamp until which the given inclusion bounds will stay in
+  // effect.
+  //
+  // Inclusion bounds refer to the range of values that are allowed for the
```

Comment:
a given metric may be set to?

I think we are abusing the word metric here...if we are including things like power to discharge at its not really a metric. A metric is something that you measure - if you set it it sounds more like a property.

Reply:
Done